### PR TITLE
gitmodules: Use non-shallow module for cl-colors2.

### PR DESCRIPTION
**Pull request by Ambrevar**
# Description

Should fix the CI build.

# Discussion

WARNING TO ALL: Notabug does not advertize commits, so when you add a gitmodule from there, **do not pass** `--depth=1`.

# Checklist:
Everything in this checklist is required for each PR.  Please do not approve a PR that does not have all of these items.

- [x] Git hygiene:
  - I have pulled from master before submitting this PR
  - There are no merge conflicts.
- [x] I've added the new dependencies as:
  - ASDF dependencies,
  - Git submodules,
    ```sh
	cd /path/to/nyxt/checkout
    git submodule add https://gitlab.common-lisp.net/nyxt/py-configparser _build/py-configparser
    ```
  - and Guix dependencies.
- [x] My code follows the style guidelines for Common Lisp code. See:
  - [Norvig & Pitman's Tutorial on Good Lisp Programming Style (PDF)](https://www.cs.umd.edu/~nau/cmsc421/norvig-lisp-style.pdf)
  - [Google Common Lisp Style Guide](https://google.github.io/styleguide/lispguide.xml)
- [x] I have performed a self-review of my own code.
- [ ] My code has been reviewed by at least one peer.  (The peer review to approve a PR counts.  The reviewer must download and test the code.)
- [x] Documentation:
  - All my code has docstrings and `:documentation`s written in the aforementioned style.  (It's OK to skip the docstring for really trivial parts.)
  - I have updated the existing documentation to match my changes.
  - I have commented my code in hard-to-understand areas.
  - I have updated the `changelog.lisp` with my changes if it's anything user-facing (new features, important bug fix, compatibility breakage).
  - I have added a `migration.lisp` entry for all compatibility-breaking changes.
  - (If this changes something about the features showcased on Nyxt website) I have these changes described in the new/existing article at Nyxt website or will notify one of maintainters to do so.
- [ ] Compilation and tests:
  - My changes generate no new warnings.
  - I have added tests that prove my fix is effective or that my feature works.  (If possible.)
  - I ran the tests locally (`(asdf:test-system :nyxt)` and `(asdf:test-system :nyxt/gi-gtk)`) and they pass.

### CODE REVIEW

**Comment by aadcg** [comment]
If it fixes the CI, then it's a splendid catch! 

How come I don't see it misbehaving locally when passing `shallow=true`? Strange...

**Comment by Ambrevar** [comment]
Because you've got the submodule in your `.git/modules` already.

To test this properly:

```sh
git clone --local /path/to/your/nyxt nyxt-test
cd nyxt-test
# your submodule tests here
```

**Comment by Ambrevar** [comment]
OK to merge?

**Review by aartaka — APPROVED** [review]

File: .gitmodules [hunk 1]
```diff
@@ -593,11 +593,11 @@
...
 	shallow = true
-[submodule "_build/cl-colors2"]
-	path = _build/cl-colors2
-	url = https://notabug.org/cage/cl-colors2
-	shallow = true
 [submodule "_build/prompter"]
 	path = _build/prompter
 	url = https://github.com/atlas-engineer/prompter
 	shallow = true
+[submodule "_build/cl-colors2_fixed"]
```
Comment (aadcg) [inline .gitmodules:600]: The suffix in the name was left on purpose?

Comment (Ambrevar) [reply .gitmodules:600]: Apparently Git does not support overriding submodules (with the same name).  So I had to use another name.
This is so dumb, but I can't find a way around this.  Suggestions welcome!

**Pull request merged**